### PR TITLE
Added a little trick to make work imgpaste with prosemirror plugin.

### PR DESCRIPTION
**Pull request by pv451**
Added a little trick to make work imgpaste with prosemirror plugin. It is raw, but it`s working.

### CODE REVIEW

**Comment by aov-arnold** [comment]
@splitbrain Could you merge this? Currently we have to edit the script.js after each update.

File: script.js [hunk 1]
```diff
@@ -38,7 +38,25 @@ jQuery(function () {
...
+                        {
+                            var sel, range;
+                            if (window.getSelection) {
+                                sel = window.getSelection();
+                                if (sel.getRangeAt && sel.rangeCount) {
+                                    range = sel.getRangeAt(0);
+                                    range.deleteContents();
+
+                                    var el = document.createElement("img");
+                                    el.innerHTML = '<img src="/lib/exe/fetch.php?media=' + data.id+'"  class="media"  draggable="true" contenteditable="false">';
```
Comment (splitbrain) [inline script.js:51]: This is an odd way to construct an img element and might be prone to XSS insertion. It should be changed to add the appropriate attributes to the `el` object.

**Review by splitbrain — CHANGES_REQUESTED** [review]

**Comment by splitbrain** [comment]
I am closing this since a new PR is required for the refactored code

**Comment by pv451** [comment]
> I am closing this since a new PR is required for the refactored code

So, sgould I create new PR, or you will create proper solution?

**Comment by splitbrain** [comment]
@pv451 a proper solution using a Prosemirror Transaction would be preferred, but TBH I'm not 100% sure how and will probably not find the time to look into it in the near future. So yeah an adjusted PR with your approach would be better than nothing I guess.

File: script.js [hunk 2]
```diff
@@ -38,7 +38,29 @@ jQuery(function () {
...
+
+                                    var el = document.createElement("img");
+                                    el.src='/lib/exe/fetch.php?media=' + data.id;
+                                    el.className="media";
+                                    el.draggable="true";
+                                    el.contentEditable="false";
+                                    range.insertNode(el);
+                                }
+                            } else if (document.selection && document.selection.createRange) {
+                                document.selection.createRange().text = '{{:' + data.id + '}}';
```
Comment (splitbrain) [inline script.js:58]: Another question: what is this else branch for? Where would this syntax be inserted?

Comment (pv451) [reply script.js:58]: To native DokuWiki editor. Actually it was in original code.

So we defines is it prosemirror and if not just insert markup as it was originally.

Comment (pv451) [reply script.js:58]: Oh, now i get it. I`ll look at my production. It wat 2 years ago, really thought this request died already.

**Pull request closed without merging**